projects/kit/components/badged-content/badged-content.component.ts

```diff
@@ -77,6 +77,14 @@ export class TuiBadgedContentComponent {
         return `0 0 0 ${borderWidth}px`;
     }
 
+    get computedColorTop(): string {
+        return this.colorTop || 'var(--tui-success-fill)';
```

Comment:
What do you think about move `'var(--tui-success-fill)'`, `'var(--tui-success-fill)'` variables to global options? Then any developers will can override in `forRoot` or `forFeature` these parameters.

Reply:
First of all, we can just move those to default values instead of empty strings. And second, yes, this falls under the task where we move default values to DI tokens.

Reply:
No, if I just move those to default value instead of empty string, colored circles will always be displayed.  https://taiga-ui.dev/components/badged-content/API?tuiMode=null&colorBottom=black&colorTop=black 
We need default value only for icons

I propose to move it to TUI_BADGED_CONTENT_OPTIONS  in a separate PR along with `size` and `rounded` as part of this task https://github.com/TinkoffCreditSystems/taiga-ui/issues/133

Reply:
I think he mean 

```ts
    @Input()
    @tuiDefaultProp()
    colorTop = 'var(--tui-success-fill)'; // TODO: move to TUI_BADGED_CONTENT_OPTIONS

    // ...
```

Reply:
I got it. I mean, there is a getter in component
```    
get bottomNotification(): string {
        return !this.contentBottom && this.colorBottom ? this.colorBottom : '';
}
```
If content is undefined and color is defined we show small circle. So we need to keep empty default colorTop and colorBottom, overwise we will see circles by default

Reply:
I don't think we need a TODO, we will eventually go over all components anyway

Reply:
I remade with default value in icon class